iam/package.json

```diff
@@ -29,17 +30,24 @@
     "luxon": "^2.4.0",
     "tslint": "^6.1.3",
     "twitter-api-sdk": "1.0.6",
-    "typescript": "~4.6.3"
+    "typescript": "~4.6.3",
+    "uuid": "^8.3.2"
```

Comment:
np: `uuid` unused, and `@ethersproject/base64`, `@types/uuid`, `uuid` in devdependencies